modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -1532,18 +1532,24 @@
      },
      "ACTIONS": {
        "SPLIT": "Split",
-       "CUT":   "Cut",
-       "REPLAY_SEGMENT": "Replay segment",
-       "REPLAY_END_OF_SEGMENT": "End of segment",
+       "SPLIT_TOOLTIP": "Create a new trim mark at the current cursor position.",
```

Comment:
Maybe "split mark" instead of "trim mark" since we don't necessarily trim? But I'm kinda unsure about this one, I don't know how prevalent "trim mark" is in video editing.

Reply:
Changed the wording. "Trim mark" is used no longer.